### DISCUSSION TITLE
[绳匠入驻]为我自己打响广告

### DISCUSSION THREAD

**Discussion by MareSera** [share121/inter-knot#1634] [其他]
你好！我是汐落！一个奇奇怪怪什么都会一点点的高中生！

[我的个人主页：[点击传送](https://www.maresera.top/)

我的博客：[点击传送](https://blog.maresera.top/)

![BG](https://github.com/user-attachments/assets/5759476e-817f-4507-add9-1acde72b48be)

emmmm接委托啦，什么委托都可以的，我自己研究过**HDD**这个东西！用C++自己写了一个（自豪）

**Comment by MareSera** [comment]
要发委托去我主页看我邮箱啊！

**Comment by capacitor1** [comment]
布什·隔门

🤔怎么都用这个frosti

搞的我也想整一个了

**Comment by Buer-Nahida** [comment]
同款Astro主题
